### PR TITLE
Fix shape function for argmax

### PR DESCRIPTION
**Pull request by qingyunqu**
Fixes #ISSUE_NUMBER

Fix shape function of argmax on `dim==None` and `keepdim==True`.

### CODE REVIEW

File: torch/jit/_shape_functions.py [hunk 1]
```diff
@@ -1070,7 +1070,11 @@ def argmax(
     self: List[int], dim: Optional[int] = None, keepdim: bool = False
 ) -> List[int]:
     if dim is None:
-        return []
+        out: List[int] = []
```
Comment (soulitzer) [inline torch/jit/_shape_functions.py:1073]: How about `return [1] * len(self) if keepdim else []`?

**Review by soulitzer — COMMENTED** [review]
Thanks for the fix. Added a small comment. Could you add a test?

**Comment by qingyunqu** [comment]
> Thanks for the fix. Added a small comment. Could you add a test?

Done

**Review by soulitzer — APPROVED** [review]
Thanks!

**Comment by soulitzer** [comment]
Test failures look real...

**Pull request closed without merging**